src/main/java/com/google/sps/data/Recipe.java

```diff
@@ -2,30 +2,41 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.time.format.DateTimeFormatter;  
-import java.time.LocalDateTime;
-import java.time.format.FormatStyle;
 
 // Class representing a Recipe
 public class Recipe {
-  // it's not done yet and I'm sure I'm going to add a bunch more methods but here's what I have so far
   private long id;
   private String recipeName;
   private long authorID;
+  // image is not implemented yet
   private String imageUrl;
+  private String description;
   private ArrayList<String> tags;
   private ArrayList<String> ingredients;
   private ArrayList<String> steps;
   private int popularity;
 
-  public Recipe(String recipeName, ArrayList<String> tags, ArrayList<String> ingredients, ArrayList<String> steps) {
+  public Recipe(long id, String recipeName, String description, ArrayList<String> tags, 
+  ArrayList<String> ingredients, ArrayList<String> steps) {
+    this.id = id;
     this.recipeName = recipeName;
+    this.description = description;
     this.tags = tags;
     this.ingredients = ingredients;
     this.steps = steps;
     popularity = 0;
   }
 
+  public Recipe(long id, String recipeName, String description, ArrayList<String> tags, 
```

Comment:
Do you need to set the description below?

Also, do you need 2 separate constructors here and above? They seem the same except for this one takes popularity as a parameter. Could you delete the one above, and just use this one, when you don't have popularity you can just pass 0?

Reply:
yeah I'll do that

Reply:
what do you mean by set the description though?

Reply:
Oh I just meant I think you're missing `this.description = description`?

Reply:
oh lol yeah